doc/connectivity/bluetooth/api/mesh/sar_cfg.rst

```diff
@@ -0,0 +1,243 @@
+
+Never more than one segmented message is transmitted at the time, but the Bluetooth mesh stack allows you to define buffers that will queue a defined
+number of ingoing or outgoing segmented messages.
+Use Kconfig options :kconfig:option:`CONFIG_BT_MESH_TX_SEG_MSG_COUNT` and :kconfig:option:`CONFIG_BT_MESH_RX_SEG_MSG_COUNT` to set the maximum number
+of queued outgoing and incoming segmented messages respectively.
+
+Incoming and outgoing segmented messages share the same segment buffer, which means that they allocate their segments from the same pool.
+Both incoming and outgoing messages allocate segments at the start of the transaction, but release them at the different time.
+While the outgoing segmented message releases its segments one by one as soon as they are acknowledged by the receiver, the incoming message will not
+release the segments until the message is fully received.
+Keep this in mind when defining the size of the buffers.
+
+SAR does not impose extra overhead on the access layer payload per segment.
+
+Segmentation and reassembly (SAR) Configuration models are foundation models defined by the
+`Bluetooth Mesh Protocol Specification version 1.1 <https://www.bluetooth.com/specifications/mesh-specifications/>`_.
+These models support the configuration of SAR behavior for nodes in a mesh network.
+
+The SAR Configuration models are:
+
+.. toctree::
+   :maxdepth: 1
+
+   sar_cfg_cli
+   sar_cfg_srv
+
+Intervals, timers and counters
+******************************
+
+Transmission of segments is separated by a segment transmission interval (see the `SAR Segment Interval Step`_ state).
+Other time intervals and delays available for the segmentation and reassembly are:
+
+* Interval between unicast retransmissions (see the states `SAR Unicast Retransmissions Interval Step`_ and `SAR Unicast Retransmissions Interval Increment`_).
+* Interval between multicast retransmissions (see the `SAR Multicast Retransmissions Interval Step`_ state).
+* Segment reception interval (see the `SAR Receiver Segment Interval Step`_ state).
+* Acknowledgment delay increment (see the `SAR Acknowledgment Delay Increment`_ state)
+
+When the last segment marked as unacknowledged is transmitted, the lower transport layer starts a retransmissions timer.
+The initial value of the SAR Unicast Retransmissions timer depends on the value of the TTL field of the message.
+If the TTL field value is greater than ``0``, the initial value for the timer is set according to the following formula::
+
+[unicast retransmissions interval step + unicast retransmissions interval increment * (TTL - 1)]
+
+If the TTL field value is ``0``, the initial value of the timer is set to the unicast retransmissions interval step.
+
+The initial value of the SAR Multicast Retransmissions timer is set to the multicast retransmissions interval.
+
+When the lower transport layer receives a segment of a segmented message, it starts an SAR Discard timer.
```

Comment:
```suggestion
When the lower transport layer receives a message segment, it starts a SAR Discard timer.
```

Reply:
It depends on whether you pronounce SAR as [sɑr] or [es/eɪ/ar/]. Not sure which one should be used.

Reply:
I believe [sar] is the most common but this is just nitpicking :)